pydy/codegen/ode_function_generators.py

```diff
@@ -641,16 +641,26 @@ class CythonODEFunctionGenerator(ODEFunctionGenerator):
 
     def __init__(self, *args, **kwargs):
 
+        self._options = {'tmp_dir': None,
+                         'prefix': 'pydy_codegen',
+                         'cse': True,
+                         'verbose': False}
```

Comment:
This `_options` dictionary could probably be built by inspecting the methods of the `CythonMatrixGenerator`, but I didn't try to figure that out.